.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
```suggestion
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        os: [ubuntu-latest, windows-latest, macos-latest]
```

i tested on ubuntu (the `cibuildwheel` used docker images).
unsure of the other two OS's.